### ISSUE TITLE
Crash when separating incoming sms into individual messages

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#247]
```
What steps will reproduce the problem?
1. Set incoming sms notification to separate messages or both
2. Receive incoming sms
3. Force closes gtalksms

What is the expected output?
Receive SMS without crash

What do you see instead?
Force closes gtalksms. message never forwards to gtalk

Additional informations:
Phone model: HTC Incredible 2, Cyanogenmod 7.1

```

Original issue reported on code.google.com by `rondepas...@gmail.com` on 3 Feb 2012 at 4:12

Attachments:
- [issuelog.txt](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-247/comment-0/issuelog.txt)

**Comment by GoogleCodeExporter** [comment]
```
Thanks for your detailed report.
```

Original comment by `fschm...@gmail.com` on 3 Feb 2012 at 5:52
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 20ee910e69ed.
```

Original comment by `fschm...@gmail.com` on 3 Feb 2012 at 6:24
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
When can we expect an update with this and other fixes?
```

Original comment by `rondepas...@gmail.com` on 16 Feb 2012 at 7:01

**Comment by GoogleCodeExporter** [comment]
```
I hope before the end of the month.
I'm trying to fix another issue but I've got only few time to work on it.

Meanwhile I will do a beta version.
```

Original comment by `Florent....@gmail.com` on 16 Feb 2012 at 9:36

**Comment by GoogleCodeExporter** [comment]
```
Beta version works too. I'm always happy to help test. I Do a little 
development on my own. Thanks for your response and keep up the great work. 
Very practical and helpful app you have here.
```

Original comment by `rondepas...@gmail.com` on 16 Feb 2012 at 9:40

**Comment by GoogleCodeExporter** [comment]
```
Thanks, it's nice to ear that!
Beta 3.4.1 available in the download section.
```

Original comment by `Florent....@gmail.com` on 16 Feb 2012 at 10:57

**Comment by GoogleCodeExporter** [comment]
```
Beta 3.4.1: Following error in Google Talk Client upon SMS receipt:


ACTION_SMS_RECEIVED - Error writing to MUC:
-- caused by: java.lang.IllegalArgumentException: Field not found for the 
specified variable name.
*SMS from ***********: * MESSAGE TEXT
Incoming SMS: Unable to send conference room configuration form. Ex: null

```

Original comment by `rondepas...@gmail.com` on 17 Feb 2012 at 4:27

**Comment by GoogleCodeExporter** [comment]
```
This means that GTalkSMS was unable to send the configuration to the MUC room. 
We can not fix that, because it's a server issue. Try to find a conference 
server that provides more XMPP MUC features.
```

Original comment by `fschm...@gmail.com` on 17 Feb 2012 at 10:30

**Comment by GoogleCodeExporter** [comment]
```
Yes I forgot to mention that we add a new setting:
1 - Set a MUC server with value like "conference.jwchat.org"
    Menu > Connection > Advance parameters > MUC Service
2 - Check the setting
    Menu > Connection > Advance parameters > Enforce this MUC Server
```

Original comment by `Florent....@gmail.com` on 18 Feb 2012 at 12:37

**Comment by GoogleCodeExporter** [comment]
```
How should I choose a MUC server? I used several and my client gives me a 
"Cannot join group chat" error every time. Thanks!
```

Original comment by `rondepas...@gmail.com` on 23 Feb 2012 at 8:18

**Comment by GoogleCodeExporter** [comment]
```
I'm also having trouble with this issue.  Not sure how to find a reliable MUC 
server
```

Original comment by `soccermo...@gmail.com` on 28 Mar 2012 at 6:38

**Comment by GoogleCodeExporter** [comment]
```
Do you try "conference.jwchat.org" with checking option "Enforce this MUC 
Server"?
```

Original comment by `Florent....@gmail.com` on 1 Apr 2012 at 2:13

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 3.5
```

Original comment by `Florent....@gmail.com` on 2 Apr 2012 at 9:38
- Changed state: **Deployed**
